spec/unit/classes/server/contrib_spec.rb

```diff
@@ -40,21 +43,4 @@
       is_expected.to contain_package('postgresql-contrib').with(tag: 'puppetlabs-postgresql')
     end
   end
-
-  describe 'on Gentoo' do
-    let :facts do
-      {
-        os: {
-          family: 'Gentoo',
-          name: 'Gentoo',
-        },
-      }
-    end
-
-    it 'fails to compile' do
-      expect {
-        is_expected.to compile
-      }.to raise_error(%r{is not supported})
-    end
```

Comment:
Can you modify this to test the package is not defined (which is the new implementation)? You could do the same for Debian.
```ruby
it { is_expected.to compile }
it { is_expected.not_to contain_package('postgresql-contrib')
```

Reply:
Done 6d7ca60